recipe/meta.yaml

```diff
@@ -77,7 +77,10 @@ about:
   home: http://openmm.org
   license: MIT
```

Comment:
Should we include this here somehow?

Reply:
Is it valid to list multiple licenses? I thought `conda-build` was pretty finicky about what it allows here.

Reply:
I think you are thinking of `license_family`.

Reply:
Ah! So can we just list as
```YAML
licenses:
- MIT
- LGPL
- GPL
```
etc?

Reply:
No, but it's better to add the most restrictive license, which is GPL and add a note in the description that if OpenCL and CUDA parts are not used, it is MIT.

Reply:
We don't actually have any GPL code.  LGPL is the most restrictive.  We have both files because LGPL isn't a self-contained license.  It incorporates the GPL by reference then adds a few exceptions to it.

Reply:
What does that mean we should include for `license_file`, then? Should we include only the LGPL text, or also GPL text?

Reply:
```
  license: LGPL-3.0-or-later
  license_family: LGPL
  license_file:
    - docs-source/licenses/Licenses.txt
    - docs-source/licenses/LGPL.txt
    - docs-source/licenses/GPL.txt
```

You can optionally add a note in `description` about openmm being mostly MIT.

Reply:
Both files are required.  The first sentence of the LGPL license is, "This version of the GNU Lesser General Public License incorporates the terms and conditions of version 3 of the GNU General Public License, supplemented by the additional permissions listed below."  If you only have the LGPL text, you're missing most of the license.

Reply:
It might make sense to say something like, MIT with some portions (fill in what this is) under LGPL.

Reply:
Addressed in last commit 727c6b4. Let me know if that's alright!